records_mover/url/base.py

```diff
@@ -75,7 +103,7 @@ def copy_to(self, other_loc: 'BaseDirectoryUrl') -> 'BaseDirectoryUrl':
         return other_loc
 
     def is_directory(self) -> bool:
-        "Returns true"
+        "Returns true"  # TODO document
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker